src/Microsoft.PowerShell.ConsoleHost/host/msh/CommandLineParameterParser.cs

```diff
@@ -195,6 +194,53 @@ internal static int MaxNameLength()
             "workingdirectory"
         };
 
+        /// <summary>
+        /// These represent the parameters that are used when starting pwsh.
+        /// We can query in our telemetry to determine how pwsh was invoked.
+        /// </summary>
+        [Flags]
+        internal enum ParameterBitmap : long
+        {
+            Command           = 0x00000001, // -Command | -c
+            ConfigurationName = 0x00000002, // -ConfigurationName | -config
+            CustomPipeName    = 0x00000004, // -CustomPipeName
+            EncodedCommand    = 0x00000008, // -EncodedCommand | -e | -ec
+            ExecutionPolicy   = 0x00000010, // -ExecutionPolicy | -ex | -ep
+            File              = 0x00000020, // -File | -f
+            Help              = 0x00000040, // -Help, -?, /?
+            InputFormat       = 0x00000100, // -InputFormat | -inp | -if
+            Interactive       = 0x00000200, // -Interactive | -i
+            Login             = 0x00000400, // -Login | -l
+            MTA               = 0x00000800, // -MTA
+            NoExit            = 0x00001000, // -NoExit | -noe
+            NoLogo            = 0x00002000, // -NoLogo | -nol
+            NonInteractive    = 0x00004000, // -NonInteractive | -noni
+            NoProfile         = 0x00008000, // -NoProfile | -nop
+            OutputFormat      = 0x00010000, // -OutputFormat | -o | -of
+            SettingsFile      = 0x00020000, // -SettingsFile | -settings
+            SSHServerMode     = 0x00040000, // -SSHServerMode | -sshs
+            STA               = 0x00080000, // -STA
+            Version           = 0x00100000, // -Version | -v
+            WindowStyle       = 0x00200000, // -WindowStyle | -w
+            WorkingDirectory  = 0x00400000, // -WorkingDirectory | -wd
+            // Parameter values for ExecutionPolicy
```

Comment:
I don't think this comment is really necessary

```suggestion
            // Enum values for ExecutionPolicy
```